static/app/views/issueDetails/actions/index.tsx

```diff
@@ -374,38 +376,42 @@ export function Actions(props: Props) {
           <ResolvedActionWapper>
             <ResolvedWrapper>
               <IconCheckmark />
-              {isResolved ? t('Resolved') : t('Archived')}
+              {isResolved ? resolvedCopyCap || t('Resolved') : t('Archived')}
```

Comment:
would there be any way to address translations of the custom copy?

Reply:
updated